fastly/fastly_test.go

```diff
@@ -22,6 +22,9 @@ var testServiceID = serviceIDForTest()
 // Default ID of the testing service.
 var defaultTestServiceID = "7i6HN3TK9wS159v2gPAZ8A"
 
+var ServiceTypeVCL = "vcl"
+var ServiceTypeWasm = "wasm"
```

Comment:
These can be constants.

```suggestion
const ( 
    // ServiceTypeVCL is the type for VCL services.
    ServiceTypeVCL = "vcl"
    // ServiceTypeWasm is the type for Wasm services.
    ServiceTypeWasm = "wasm"
)
```